boot/seal.go

```diff
@@ -217,9 +225,9 @@ func resealKeyToModeenv(rootdir string, model *asserts.Model, modeenv *Modeenv,
 		return fmt.Errorf("cannot prepare for key resealing: %v", err)
 	}
 	resealKeyParams := &secboot.ResealKeyParams{
-		ModelParams:             modelParams,
-		KeyFile:                 filepath.Join(InitramfsEncryptionKeyDir, "ubuntu-data.sealed-key"),
-		TPMPolicyUpdateDataFile: filepath.Join(dirs.SnapFDEDirUnder(rootdir), "policy-update-data"),
+		ModelParams: modelParams,
+		KeyFile:     filepath.Join(InitramfsEncryptionKeyDir, "ubuntu-data.sealed-key"),
+		AuthKey:     tpmPolicyAuthKey,
```

Comment:
shall we verify that the policy auth key has been set?

Reply:
ResealKey() tests it and returns an error if it's unset.